<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_video</class>
 <widget class="QDialog" name="add_video">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adicionar Vídeo</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>351</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Título </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_titulo"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Tipo : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_tipo"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Faixa Etária</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="video_age"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Valor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="video_valor"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>ID Vídeo :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="video_id"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>ID Alugando :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="video_idalug">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="btn_imagem">
        <property name="text">
         <string>Buscar Imagem (Capa)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_salvar">
        <property name="text">
         <string>Salvar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
